draft-ietf-rats-eat.md

```diff
@@ -1208,21 +1214,22 @@ security state of the entity storing the private key used in a PoP application.
 
 # Detached EAT Bundles {#DEB}
 
-A detached EAT bundle is a structure to convey a fully-formed and signed token plus detached claims set that relate to that token.
-It is a top-level EAT message like a CWT or JWT.
-It can be occur any place that CWT or JWT messages occur.
-It may also be sent as a submodule.
+A detached EAT bundle is a structure to convey an EAT plus detached claims sets secured by that EAT.
+It is a top-level message like a CWT or JWT.
+It can be occur any place that a CWT or JWT occurs.
+It may be sent as a submodule.
 
 A detached EAT bundle consists of two parts.
 
-The first part is a full top-level token.
-This top-level token MUST have at least one submodule that is a detached digest.
-This top-level token may be either CBOR or JSON-encoded.
+The first part is an encoded EAT.
+This EAT MUST have at least one submodule that is a detached digest.
+This EAT may be either CBOR or JSON-encoded.
```

Comment:
Is isn't clear if the detatched claims set must be encoded using the same encoding as was used for the bounding EAT. Can the detatched EAT bundle be JSON encoded, while the EAT containing the digest of the bundle be CBOR encoded?

Reply:
Added a comment that the bundle can be either CBOR or JSON which should clarify that mixing is allowed.

The mixing of encoding is very explicitly allowed here to accommodate composite devices where you don't control the serialization format of attesters in the components used to make up the composite device.